### DISCUSSION TITLE
Welcome to Overseerr Discussions!

### DISCUSSION THREAD

**Discussion by sct** [sct/overseerr#215] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dedepene** [comment]
Great work, @sct - keep it up. This is already shaping up as a pretty sweet addition to my plex setup :) 🐱‍🏍

**Reply by sct** [reply]
Thank you! Appreciate the kind words!
